README.md

```diff
@@ -141,9 +168,10 @@ All grasshopper services and apps can be built as [Docker](https://docs.docker.c
     }
     ```
 
-    **Note:** If using `boot2docker`, the following with get you the  `docker-provided-ip`:
+    Other URLs:
 
-        $ boot2docker ip
+    - UI = http://{{docker-provided-ip}}
+    - Geocoder = http://{{docker-provided-ip}}/geocode/{{address}}
```

Comment:
I think this should be: `http://{{docker-provided-ip}}/geocoder/geocode/{{address}}`.